mu-ruby-template.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+cd /usr/src/app
+if [ "$RACK_ENV" == "production" ];
+then
+  exec ruby $RUBY_OPTIONS $APP_ENTRYPOINT
+else
+  bundle install
+  if [ "$RACK_ENV" == "test" ];
+  then
+    rspec
```

Comment:
shouldn't this be 
```suggestion
    bundle exec rspec
```

inline with the jruby shell script